pkg/server/commands/list_objects.go

```diff
@@ -406,20 +409,16 @@ func (q *ListObjectsQuery) Execute(
 
 	objects := make([]string, 0)
 
-	var errs *multierror.Error
+	var skippableErrors *multierror.Error
 
 	for result := range resultsChan {
 		if result.Err != nil {
 			if errors.Is(result.Err, serverErrors.AuthorizationModelResolutionTooComplex) {
 				return nil, result.Err
 			}
 
-			if errors.Is(result.Err, condition.ErrEvaluationFailed) {
-				errs = multierror.Append(errs, result.Err)
-				continue
-			}
-
-			if errors.Is(result.Err, context.Canceled) || errors.Is(result.Err, context.DeadlineExceeded) {
+			if IsSkippableError(result.Err) {
+				skippableErrors = multierror.Append(skippableErrors, serverErrors.ValidationError(result.Err))
 				continue
 			}
```

Comment:
`if errors.Is(result.Err, context.Canceled) || errors.Is(result.Err, context.DeadlineExceeded) { continue } ` unreachable and uncovered 
<img width="1156" alt="image" src="https://github.com/openfga/openfga/assets/5374887/fbe2aae9-e73b-4738-8840-2d0ffa211603">